scipy/special/_logsumexp.py

```diff
@@ -155,16 +154,15 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
     return (out, sgn) if return_sign else out
 
 
-def _wrap_radians(x, xp=None):
-    xp = array_namespace(x) if xp is None else xp
```

Comment:
At some point it was encouraged to do this. I don't know why, but I followed along. I don't see why private functions shouldn't insist on being passed `xp`, so this LGTM.